utils/MoveType.json

```diff
@@ -1,5 +1,5 @@
 {
-  "INITIALIZE": "initialize",
+  "DEAL": "deal",
```

Comment:
Renamed `initialize` to `deal` to be more thematically consistent with the idea that p1 is the dealer and the first gamestate happens when p1 deals.